include/qpdf/QPDFObjectHandle.hh

```diff
@@ -1610,6 +1606,22 @@ class QPDFObjectHandle::QPDFArrayItems
     QPDFObjectHandle oh;
 };
 
+namespace qpdf
```

Comment:
(no action required on this comment)

It took me a while to figure out what was going on here. I guess these have to be in this header instead of ObjectHandle.hh because they access private constructors. I didn't even know you could do this.

I'm okay with the use of the `qpdf` namespace. C++ didn't have namespaces when I first made qpdf in 2002, and I didn't switch to use them in 2008 for the first public release, unfortunately, but there's no reason we can't use them now.